src/definitions/behaviors/form.js

```diff
@@ -730,42 +730,53 @@
                                         var date = $calendar.calendar('get date');
 
                                         if (date !== null) {
-                                            if (settings.dateHandling == 'date') {
-                                                values[name] = date;
-                                            } else if (settings.dateHandling == 'input') {
-                                                values[name] = $calendar.calendar('get input date');
-                                            } else if (settings.dateHandling == 'formatter') {
-                                                var type = $calendar.calendar('setting', 'type');
+                                            switch (settings.dateHandling) {
+                                                case 'date': {
```

Comment:
Extra braces again? What for? There is no extra let/const scope necesary (applies to all other cases)

Reply:
braces in `case` block everywhere, like in `if`, asserted by default/recommended Unicorn eslint config

Reply:
i dont agree as long as is not really necessary because of scoping. I know you can do it almost everywhere, but without a  need for scoping, whats the point? just to satisly the linter? No. We can add braces once they are really necessary (you havent had them before and without them is valid syntax since ages.